src/sacn/receiver.c

```diff
@@ -774,24 +786,35 @@ etcpal_error_t assign_receiver_to_thread(SacnReceiver* receiver, const SacnRecei
 
   SACN_ASSERT(assigned_thread);
 
-  etcpal_error_t res = sacn_add_receiver_socket(receiver->thread_id, kEtcPalIpTypeV4, config->universe_id,
-                                                receiver->netints, receiver->num_netints, &receiver->ipv4_socket);
-  if (res == kEtcPalErrNoNetints)
-    res = kEtcPalErrOk;  // Try IPv6.
+  etcpal_error_t res = kEtcPalErrOk;
+
+  if ((receiver->ip_supported == kSacnIpV4Only) || (receiver->ip_supported == kSacnIpV4AndIpV6))
```

Comment:
This if clause and the next could be turned into a single utility function.  It's used here and above.